README.md

```diff
@@ -427,6 +428,48 @@ var signIn = new OktaSignIn(
 );
 ```
 
+### showSignIn
+
+> **Note**:
+ The showSignIn method is backward compatible. You can use it with both Okta Identity Engine (as of Widget v5.5.0) and Okta Classic Engine.
+
+Recommended for most use cases. WEB applications should use the [showSignInAndRedirect](#showsigninandredirect) method.
+
+Receives an interaction code, either by Promise or redirect, depending on [sign-in policy](https://help.okta.com/oie/en-us/Content/Topics/identity-engine/procedures/set-up-default-app-redirect.htm). Renders the widget to the DOM to prompt the user to sign in. Results in one of the following:
```

Comment:
We should not mention interaction code here. This method works for both V1 and OIE. The response object that it returns will vary based on which pipeline it uses.  The V1 response will have a "status", ad if status = "SUCCESS" then it will have a method `res.session.setCookieAndRedirect` (we have this all documented somewhere around here). There are other possible statuses, mostly related to MFA

If running on OIE, the response will contain tokens (errors will be on the Promise reject or "error" callback).

The response/errors are the same as those from `renderEl`